src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -53,9 +53,15 @@ New Relic has several charts for the different components which offer different
 * [`nri-metadata-injection`](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection): Sets up a minimal `MutatingAdmissionWebhook` that injects a couple of environment variables in the containers. These contain metadata about the cluster and New Relic installation and will be later picked up by applications instrumented using APM, allowing to [correlate APM and infrastructure data](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes/).
 * [`nri-statsd`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-statsd): New Relic StatsD integration.
 
-For convenience, New Relic provides the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart, which pulls a selectable set of the charts mentioned above. `nri-bundle` can also install Kube State Metrics and Pixie for you if needed.
+Although you can install these components separately, we strongly recommend you use the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. New Relic provides this chart, which acts as a wrapper or a meta-package for the individual charts mentioned above. The use of this chart enables you these advantages: 
 
-While it's possible to install those charts separately, we strongly recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart for Kubernetes deployments, as it ensures that values across all the charts are consistent and provides full control over which components are installed, as well as the possibility to configure all of them as [Helm dependencies](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart). This is the same chart that is used and referenced by our [Kubernetes guided install](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure#guided-install).
+- It provides full control over which components are installed.
+- It ensures that their installed versions are compatible with each other.
+- It ensures that their configuration values are consistent across the installed charts.
+
+Using the `nri-bundle` chart, each component is installed as a separate [Helm dependency](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart). You can configure them individually using the parameters mentioned [here](#configure).
```

Comment:
```suggestion
- It ensures that their configuration values are consistent across the installed charts.
- Using the `nri-bundle` chart, each component is installed as a separate [Helm dependency](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart). You can configure them individually using the parameters mentioned [here](#configure).
```

Reply:
I've added the "Using..." paragraph with the first bullet that refers to components as well.